src/ar_kernel.cpp

```diff
@@ -591,23 +561,14 @@ uint32_t ar_get_system_load(void)
 // See ar_kernel.h for documentation of this function.
 uint32_t ar_get_tick_count(void)
 {
-#if AR_ENABLE_TICKLESS_IDLE
-    uint32_t elapsed_ticks = ar_port_get_timer_elapsed_us() / (kSchedulerQuanta_ms * 1000);
-    return g_ar.tickCount + elapsed_ticks;
-#else
-    return g_ar.tickCount;
-#endif // AR_ENABLE_TICKLESS_IDLE
+    return ar_port_get_time_absolute_ticks();
 }
 
 // See ar_kernel.h for documentation of this function.
 uint32_t ar_get_millisecond_count(void)
 {
-#if AR_ENABLE_TICKLESS_IDLE
-    uint32_t elapsed_ms = ar_port_get_timer_elapsed_us() / 1000;
-    return g_ar.tickCount * ar_get_milliseconds_per_tick() + elapsed_ms;
-#else
-    return g_ar.tickCount * ar_get_milliseconds_per_tick();
-#endif // AR_ENABLE_TICKLESS_IDLE
+    return ar_port_get_time_absolute_ms();
+    // return ar_port_get_time_absolute_ticks() * kSchedulerQuanta_ms; //do we have to limit resolution?
```

Comment:
Imo, it's better if resolution is not limited.

Reply:
Ok, I was not sure about consequences. Will Fix.